charts/cluster-scanner/templates/secret.yaml

```diff
@@ -14,6 +14,14 @@ data:
 {{- if and (eq .Values.imageSbomExtractor.cache.type "distributed") ((.Values.imageSbomExtractor.cache.redis).password) }}
   cache_redis_password: {{ .Values.imageSbomExtractor.cache.redis.password | b64enc | quote }}
 {{- end }}
+{{- if .Values.runtimeStatusIntegrator.natsJS.tls.enabled }}
+  {{- if eq (include "cluster-scanner.nats.tls.customTLS" .) "true" }}
+  js_tls_key_path: {{ required "please provide a TLS private key path" .Values.runtimeStatusIntegrator.natsJS.tls.key |  quote }}
+  js_tls_cert_path: {{ required "please provide a TLS certificate path" .Values.runtimeStatusIntegrator.natsJS.tls.cert | quote }}
+  {{- else }}
+  {{- include "cluster-scanner.nats.tls.certificate" . }}
+  {{- end }}
+{{- end }}
```

Comment:
Is it ok to treat paths as secrets? I don't know what the best practices are in this case, but maybe we should include here only the real secrets and move the non-secret config to a configmap. I repeat, I'm not an expert, just sharing my thoughts. cc @michele-mangili

Reply:
I agree @mech-pig